test/visualData.ts

```diff
@@ -78,6 +79,17 @@ module powerbi.extensibility.visual.test {
         public valuesStartDate = GanttData.getRandomUniqueDates(this.valuesTaskTypeResource.length, new Date(2015, 7, 0), new Date(2017, 7, 0));
         public valuesDuration = GanttData.getRandomUniqueNumbers(this.valuesTaskTypeResource.length, 3, 40);
         public valuesCompletePrecntege = GanttData.getRandomUniqueNumbers(this.valuesTaskTypeResource.length);
+        public valuesExtraInformation = GanttData.getTexts(this.valuesTaskTypeResource, "Description");
+
+        public static getTexts(valuesTaskTypeResource: string[][], text: string): string[] {
+            let result: string[] = [];
+            const count: number = valuesTaskTypeResource.length;
+            for (let i: number = 0; i < count; i++) {
```

Comment:
can we use **.map** function instead of for loop?

Reply:
Yes, sure

Reply:
Done